project/Dependencies.scala

```diff
@@ -22,6 +22,7 @@ object Dependencies {
   val workbenchGoogle2V = "0.7-f3b5a5a"
   val workbenchMetricsV = "0.3-c5b80d2"
   val workbenchNewRelicV = "0.3-8bae8e8"
+  val workbenchOpenTelemetryV = "0.1-2587881-SNAP"
```

Comment:
Moving to non-SNAP version before merging?

Reply:
For some reason my IntelliJ can't import the SNAP version  🤷‍♂

Reply:
it's not published to jfrog :)